addons/website_livechat/static/tests/new/thread_tests.js

```diff
@@ -43,7 +43,7 @@ QUnit.test("Rendering of visitor banner", async function (assert) {
         target,
         ".o_country_flag[data-src='/base/static/img/country_flags/be.png']"
     );
-    assert.containsOnce(target, "span:contains(Visitor #11)");
+    assert.containsOnce(target, "span:contains(Visitor 11)");
```

Comment:
It looks like the code is broken and not the test, supposedly this should display the name of the visitor instead of the anonymous name?